### PR TITLE
Queue capacity

### PR DESCRIPTION
**Pull request by nikunj1705**

### CODE REVIEW

File: docs/agent-chat/teams.md [hunk 1]
```diff
@@ -77,6 +77,30 @@ Similar to the Team Offline message, you can configure the delay message that is
 4. View agent status
 5. View agent chats
 
+## Queue Capacity and Online Agent slots
+
+![Queue capacity 1](assets/case_1.png)
+
+Check the image above. Here, you get a sense of the following metrics - 
+1. Chat handling capacity: This is a %age value that signifies if your team capacity is ideal or your team cannot handle the current incoming chat volume. For example, if the number value is in red and the copy states `over capacity`, it means the chat volume supercedes the available online agents' capacity 
+2. Chats waiting to be taken: Total number of chats in queue i.e. all users waiting in pending state
```
Comment (medhaanand) [inline docs/agent-chat/teams.md:86]: Provide a space between points here too as you have provided below.

Comment (medhaanand) [reply docs/agent-chat/teams.md:86]: @nikunj1705

File: docs/agent-chat/teams.md [hunk 2]
```diff
@@ -77,6 +77,30 @@ Similar to the Team Offline message, you can configure the delay message that is
 4. View agent status
 5. View agent chats
 
+## Queue Capacity and Online Agent slots
```
Comment (medhaanand) [inline docs/agent-chat/teams.md:80]: Why is this being called Queue Capacity? Isnt it Team Capacity?

File: docs/agent-chat/teams.md (hunk 1 above)
Comment (nikunj1705) [reply docs/agent-chat/teams.md:90]: done

File: docs/agent-chat/teams.md (hunk 2 above)
Comment (nikunj1705) [reply docs/agent-chat/teams.md:80]: Changed.

File: docs/agent-chat/teams.md [hunk 3]
```diff
@@ -77,6 +77,35 @@ Similar to the Team Offline message, you can configure the delay message that is
 4. View agent status
 5. View agent chats
 
+## Chat handling Capacity and Online Agent slots
```
Comment (medhaanand) [inline docs/agent-chat/teams.md:80]: This is Team Capacity in the images. Why cant we use the same name?

File: docs/agent-chat/teams.md [hunk 4]
```diff
@@ -77,6 +77,35 @@ Similar to the Team Offline message, you can configure the delay message that is
...
+
+### How is Team capacity calculated?
+
+![Queue capacity 2](assets/case_2.png)
+
+Let's understand the formula to get Chat handling capacity by a example -
+
+i. Total concurrency for online agents in the team = A
+
+ii. Waiting + Pinned Chats total for all agents in the shown team = B
```
Comment (medhaanand) [inline docs/agent-chat/teams.md:104]:
```suggestion
ii. Waiting + Assigned Chats total for all agents in the shown team = B
```

File: docs/agent-chat/teams.md [hunk 5]
```diff
@@ -77,6 +77,35 @@ Similar to the Team Offline message, you can configure the delay message that is
...
+
+### How is Team capacity calculated?
+
+![Queue capacity 2](assets/case_2.png)
+
+Let's understand the formula to get Chat handling capacity by a example -
+
+i. Total concurrency for online agents in the team = A
+
+ii. Waiting + Assigned Chats total for all agents in the shown team = B
```
Comment (medhaanand) [inline docs/agent-chat/teams.md:104]:
```suggestion
ii. Queued + Assigned Chats total for all agents in the shown team = B
```

**Pull request merged**